### PR TITLE
Add dependabot schedule for bridge / bridge-server

### LINKED ISSUE #4077: Add bridge and bridge server to dependabot
Please adapt [dependabot](https://github.com/keptn/keptn/blob/master/.github/dependabot.yml) to also consider bridge and bridge/server package.json for package updates.

### PR DESCRIPTION
**Pull request by laneli**
Closes #4077 
Signed-off-by: Elisabeth Lang <elisabeth.lang@dynatrace.com>

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -105,3 +105,13 @@ updates:
...
+  - package-ecosystem: npm
+    directory: "/bridge"
+    schedule:
+      interval: weekly
+      day: "wednesday"
+  - package-ecosystem: npm
+    directory: "/bridge/server"
+    schedule:
+      interval: weekly
+      day: "wednesday"
```
Comment (ermin-muratovic) [inline .github/dependabot.yml:117]: Is this necessary, or is it already covered by the configuration for directory `"/bridge"`?

**Review by ermin-muratovic — CHANGES_REQUESTED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply .github/dependabot.yml:117]: Subfolders that have their own package.json need to be included, so that's required.

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Pull request merged**